features/supplier/supplier_account.feature

```diff
@@ -167,6 +167,7 @@ Scenario: Supplier user can provide and change supplier details before confirmin
   And I see 'that supplier.dunsNumber' text on the page
 
 @requires-credentials @notify
+@skip-staging
```

Comment:
Isn't this scenario on the Supplier FE?

Reply:
It includes the step `Then that user can not log in using their correct password` which is user frontend.

Reply:
Ah gotcha. Good spot 😸